src/components/SchemaTable/index.jsx

```diff
@@ -27,7 +27,7 @@ const useStyles = makeStyles(theme => ({
   },
   /** Rows for the left and right panels */
   row: {
-    borderBottom: `${theme.spacing(0.25)}px solid ${theme.palette.divider}`,
+    borderBottom: `${theme.spacing(0.25)}px solid ${theme.palette.grey[300]}`,
```

Comment:
Replaced grid border colors to `theme.palette.grey` instead of `theme.palette.divider` since the theme's divider is not automated according to the color given by the user. 
Using a grey color will ensure that the grid borders are always visible since the background color of the table is always either a white or black (specifically, either #fff or rgba(0,0,0,0.87)) due to using `theme.palette.getContrastText()`.

Reply:
> since the theme's divider is not automated according to the color given by the user.

I'm not sure I understand. Any clarifications would be appreciated.

Reply:
Reversing back to using `theme.palette.divider` and letting the user config the border colors instead. See https://github.com/taskcluster/material-ui-json-schema-viewer/pull/75/files#r379640957